README.md

```diff
@@ -1,14 +1,15 @@
 # python-libprio
 [![CircleCI](https://circleci.com/gh/mozilla/python-libprio.svg?style=svg)](https://circleci.com/gh/mozilla/python-libprio)
 
-A python wrapper around libprio.
+A set of tools for aggregating data in a privacy preserving way. This repository
+includes a Python wrapper around mozilla/libprio and is designed for
+implementing server-side components.
```

Comment:
How about something like:

Prio is an algorithm for aggregating data in a privacy-preserving way. This repository includes a Python wrapper around the [libprio library](https://github.com/mozilla/libprio) and a command-line tool that can be used to batch process large amounts of input data using Prio's dual-server architecture.

For more information about Prio, see [this blog post](https://hacks.mozilla.org/2018/10/testing-privacy-preserving-telemetry-with-prio/).